README.md

```diff
@@ -195,6 +198,19 @@ async fn handle_event(
 }
 ```
 
+and in your `Cargo.toml`, include the following dependencies:
+
+```toml
+...
+[dependencies]
+futures = "0.3.16"
+tokio = { version = "1.9.0", features = ["full"] }
```

Comment:
It makes no sense to specify a minimum version of `0.3.16` for futures or `1.9.0` for tokio when features from those releases are not being used and the same isn't done for twilight. Further, enabling all features in tokio is not required.

```suggestion
futures = "0.3"
tokio = { version = "1.0", features = ["macros, rt-multi-thread"] }
```

Reply:
Good to know, I haven't done this kind of dependency precision before :100:

Reply:
Seems like I forgot some quotes :hand_over_mouth: This needs a hotfix, sorry for that!